Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build like: docker build -t converter .
+# Use like: docker run --rm -t -v $(pwd):/project -w /project converter
+
+FROM hyperized/prestissimo:latest as dependencies
+# Quickly install project dependencies with prestissimo
+WORKDIR /project
```

Comment:
Are these hard-coded values and are they somehow connected to the usage section?

```sh
docker run --rm -t -v $(pwd):/project -w /project converter
```

Does this mean we're mapping our local project to the `/project` folder on the Docker container?

Would the following work?

```sh
docker run --rm -t -v $(pwd):/path/to/local/project -w /project converter
```

Excuse the naive questions. I'm not a Docker novice.

Reply:
> Are these hard-coded values and are they somehow connected to the usage section?
> 
> ```shell
> docker run --rm -t -v $(pwd):/project -w /project converter
> ```
> Does this mean we're mapping our local project to the `/project` folder on the Docker container?

They only relate to the build process. It includes the current directory in the build process under `/project` to keep things consistent. It's not a requirement to do this, but it makes copying files across build stages easier.

> Would the following work?
> 
> ```shell
> docker run --rm -t -v $(pwd):/path/to/local/project -w /project converter
> ```

It would, but only if you point the converter to the correct path. The `-w` switch means `work directory`. So that's the path inside the container that is considered the current directory when the container is running.

You are free to mount anything into the container at any path you like (except `/`), however for sanity sake I repeated the build process where the local files are mounted into the container at the `/project` path and you set the work directory to `/project` and then if you run the converter tool it will be able to see all the files as if it's ran in the current directory.

This would work just as well:

```shell
docker run --rm -t -v $(pwd):/something/else -w /something/else converter --phpstan-json-file=phpstan.json
```
as well as this:
```shell
docker run --rm -t -v $(pwd):/something/else converter --phpstan-json-file=/something/else/phpstan.json
```

> Excuse the naive questions. I'm not a Docker novice.

Thanks for asking. It shows my PR needs explaining.
Let me know if something is unclear.